### PR TITLE
New package: freifunkMUC.FreifunkMeet version 2024.3.0

### LINKED ISSUE #147935: [Package Request]: FreifunkMeet (Jitsi electron fork)
### How can we help?

I would like help so I can submit the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/freifunkMUC/jitsi-meet-electron/releases
Publisher: Freifunk München
Package Name: FreifunkMeet (meet.ffmuc.net)
Description: Electron application for Freifunk Meet
Package Version: [v2024.3.0](https://github.com/freifunkMUC/jitsi-meet-electron/releases/tag/v2024.3.0)
Installer URL: https://github.com/freifunkMUC/jitsi-meet-electron/releases/download/v2024.3.0/ffmuc-meet.exe

We would like to add auto Publishing for the our fork of Jitsi Meet (Freifunk Meet).
Can someone help out? how should we do this?
We thought about a Github action if necessary

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] This resolves #147935

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/147987)

### CODE REVIEW

File: manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Publisher: Freifunk Muenchen
+PublisherUrl: https://github.com/freifunkMUC
+PackageName: FreifunkMeet
+PackageUrl: https://github.com/freifunkMUC/jitsi-meet-electron
+License: Apache-2.0
+LicenseUrl: https://github.com/freifunkMUC/jitsi-meet-electron/raw/main/LICENSE
+Copyright: Copyright © 2024 Freifunk Muenchen
+ShortDescription: 'Jitsi Meet desktop application powered by :electron:'
+Tags:
+- hacktoberfest
```
Comment (T0biii) [inline manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.locale.en-US.yaml:16]: should be the same as: https://github.com/microsoft/winget-pkgs/blob/master/manifests/j/Jitsi/Meet/2024.3.0/Jitsi.Meet.locale.en-US.yaml#L18-L23 ?

**Review by T0biii — CHANGES_REQUESTED** [review]

File: manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: freifunkMUC.FreifunkMeet
+PackageVersion: 2024.3.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
```
Comment (T0biii) [inline manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.installer.yaml:11]: should also be align to: https://github.com/microsoft/winget-pkgs/blob/master/manifests/j/Jitsi/Meet/2024.3.0/Jitsi.Meet.installer.yaml#L12

**Review by T0biii — CHANGES_REQUESTED** [review]

**Comment by T0biii** [comment]
we also have a privacy url, PublisherUrl, SupprotUrl: 

```
PrivacyUrl: https://ffmuc.net/privacy/
PublisherUrl: https://ffmuc.net
PublisherSupportUrl: https://github.com/freifunkMUC/jitsi-meet-electron/issues
```

File: manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.locale.en-US.yaml (hunk 1 above)
Comment (Dragon1573) [reply manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.locale.en-US.yaml:16]: Maybe it should?

Comment (Dragon1573) [inline manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.locale.en-US.yaml:16]:
```suggestion
Tags:
- chat
- jitsi
- jitsi-meet
- meeting
- video
```

File: manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.installer.yaml (hunk 2 above)
Comment (Dragon1573) [inline manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.installer.yaml:11]:
```suggestion
UpgradeBehavior: install
```

File: manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: freifunkMUC.FreifunkMeet
+PackageVersion: 2024.3.0
+PackageLocale: en-US
+Publisher: Freifunk Muenchen
+PublisherUrl: https://github.com/freifunkMUC
```
Comment (Dragon1573) [inline manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://ffmuc.net
PublisherSupportUrl: https://github.com/freifunkMuc/jitsi-meet-electron/issues
PrivacyUrl: https://ffmuc.net/privacy/
```

Add properties according to @T0biii's suggestion.

**Review by T0biii — APPROVED** [review]
LGTM now

**Review by stephengillie — APPROVED** [review]

**Pull request merged**